### ISSUE TITLE
feat: Contributing - Engine - List all major 3rd party dependencies

### ISSUE THREAD

**Issue by VaclavElias** [stride3d/stride-docs#338]
As a user, I would like to see (understand) a list of all major 3rd party dependecies.

- Bepu
- Bullet
- ..

**Comment by VaclavElias** [comment]
@Doprez I recall we have this somewhere in the issue or discussion, can you recall?

**Comment by Jklawreszuk** [comment]
@VaclavElias All dependencies are located in the deps folder. Take a look :)

**Comment by VaclavElias** [comment]
Thanks. Ok, that's going to be one source. The other, I figure out some major NuGet packages we are using/referencing.

**Comment by Jklawreszuk** [comment]
When it comes to silk wrappers and managed code - .NET, native code (mostly) - deps :v

**Comment by Doprez** [comment]
> @Doprez I recall we have this somewhere in the issue or discussion, can you recall?

The only spot I can recall is the one Jkao is talking about sorry.

**Comment by Doprez** [comment]
Or actually, I was thinking about the third party file https://github.com/stride3d/stride/blob/master/THIRD%20PARTY.md

**Comment by VaclavElias** [comment]
Yeah, that is the one I recall from somewhere! Good to know that our brains work better together 🤣

**Comment by Doprez** [comment]
lol nice! It doesnt have Bepu in there though and if we are adding that we may want to include DotRecast as well?

**Comment by VaclavElias** [comment]
Yes, I would like to re-do this page and move it to docs. So anything missing can be added to this issue and I will add it :). Thanks for `DotRecast` suggestion.
